File: Array/2270.Number of Ways to Split Array/Solution.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        s = sum(nums)
        ans = cur = 0
        for i in range(len(nums) - 1):
            cur += nums[i]
            s -= nums[i]
            if cur >= s:
                ans += 1
        return ans

File: Array/2270.Number of Ways to Split Array/Solution.java
class Solution {
    public int waysToSplitArray(int[] nums) {
        long s = 0, cur = 0;
        for (int v : nums) {
            s += v;
        }
        int ans = 0;
        for (int i = 0; i < nums.length - 1; i++) {
            cur += nums[i];
            s -= nums[i];
            if (cur >= s) {
                ans++;
            }
        }
        return ans;
    }
}